src/main/java/com/epam/spring/homework3/service/impl/HotelServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.spring.homework3.service.impl;
+
+
+import com.epam.spring.homework3.controller.dto.HotelDTO;
+import com.epam.spring.homework3.service.HotelService;
+import com.epam.spring.homework3.service.mapper.HotelMapper;
+import com.epam.spring.homework3.service.model.Hotel;
+import com.epam.spring.homework3.service.repository.HotelRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
```

Comment:
Use @Service annotation instead of @Component, it has actually no difference, but first one has more readability to reviewers